iso.bbx

```diff
@@ -309,19 +317,63 @@
         {#1}}% This is the go-to format
 }
 
+% Define names to consider for 'labelname', based on
+% the default definition, but adding holder as the
+% very first choice for patents.
+\DeclareLabelname[patent]{%
+  \field{holder}
+  \field{shortauthor}
+  \field{author}
+  \field{shorteditor}
+  \field{editor}
+  \field{translator}
+}
+
 % Define dates to consider for 'labeldate', based on
 % the default definition, but without 'urldate' field,
 % preventing printing url seen date in place of the year
 % of a publication.
 \DeclareLabeldate{%
+  \field{origdate}
   \field{date}
   \field{year}
   \field{dateaddon}% <---- biblatex-iso690 data model extension
   \field{eventdate}
-  \field{origdate}
   \literal{nodate}
 }
 
+% Sorting
+\DeclareSortingTemplate{nyt}{
```

Comment:
Ping @wg030 What is the biblatex default here?

Reply:
Well, the sorting template essentially determines the order of entries in the bibliography. I did not change much except for `holder` and `origdate`. If we did not do that here, then for our example with `holder = {{Winget Ltd}}` and `author = {Reginald John England}`, this entry would be orderd under the letter E in the bibliography altough the entry starts with a W. This would look completly weird, of course. With this sorintg template the entry would corretly appear under W